openwisp_users/models.py

```diff
@@ -25,6 +26,29 @@ class Organization(BaseOrganization, AbstractOrganization):
     class Meta(AbstractOrganization.Meta):
         swapper.swappable_setting('openwisp_users', 'Organization')
 
+    
+    def add_user(self, user, **kwargs):
+        """
+        overriding this method of ``AbstractOrganization``
+        to handle double organization owner creation
+        """
+        try:
+            org_user = super().add_user(user, **kwargs)
+        except IntegrityError as e:
+            # if ``super().add_user()`` again tries to create
+            # organization owner then catch the error and returns 
+            # the organization user created
+            if "UNIQUE constraint failed: openwisp_users_organizationowner.organization_user_id" in str(e):
```

Comment:
I am not sure this would work in all databases

Reply:
Do you want me to just except integrity error not check the exact message?

Reply:
I just thought maybe we could find a better way, see here:

https://github.com/bennylope/django-organizations/blob/2dd044956cf03dad9c8c4954ba325d2769fe3643/src/organizations/abstract.py#L96C1-L117

What if we override the `create` method of our org owner model to verify whether an owner already exists before creating a new one?

Reply:
Overriding was my first solution but I thought may be we wanted to use django-organizations methods that's why i implemented this. Let me override this and send you PR, thanks.